modules/wait-for-vmi-status/go.mod

```diff
@@ -71,7 +71,12 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace k8s.io/client-go => k8s.io/client-go v0.23.1
+// Kubernetes
+replace (
+	k8s.io/api => k8s.io/api v0.23.4
+	k8s.io/apimachinery => k8s.io/apimachinery v0.23.4
+	k8s.io/client-go => k8s.io/client-go v0.23.4
```

Comment:
I prefer to put the largest set of these replaces (in each group) to all modules even though they might not be used. This simplifies the update of modules or creating a new module to just ctrl+c and ctrl+v. Just mentioning this, but it is a matter of preference.